src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    int index = new Random().nextInt(Colors.values().length);
```

Comment:
this object will have the same number every time. Let's create a Random object here, add the private access modifier and transfer the logic to the method